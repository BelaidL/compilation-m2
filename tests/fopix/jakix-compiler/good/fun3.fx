../../interpreter/good/fun3.fx