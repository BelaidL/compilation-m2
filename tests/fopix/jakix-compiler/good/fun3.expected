../../interpreter/good/fun3.expected